crml/governance/Cargo.toml

```diff
@@ -20,6 +20,11 @@ scale-info = { version = "1.0.0", default-features = false, features = ["derive"
 log = { version = "0.4.14", default-features = false }
 sp-runtime = { git = "https://github.com/paritytech/substrate", version = "4.0.0-dev", default-features = false  }
 
+[dev-dependencies]
+crml-generic-asset = { path = "../generic-asset", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate", version = "4.0.0-dev", default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", version = "4.0.0-dev", default-features = false }
```

Comment:
for `dev-dependencies` don't need `default-features = false`, tests are ok to build with `std`